oop/src/views/YourInfoPage.vue

```diff
@@ -291,6 +291,11 @@ export default {
 </script>
 
 <style scoped>
+.phn-input, .phone-number {
+  max-width: 100%;
+  width: 350px;
+}
```

Comment:
Why are these CSS classes necessary? The PHN and phone number fields should be limited to 160px (which is already being set).